LICENSE

```diff
@@ -1,13 +1,14 @@
-Copyright 2024 SUSE LLC
+Copyright (C) 2024  SUSE LCC
```

Comment:
No need to change the header, keep it consistent among all the repos, for the yearly search&replace.

```suggestion
Copyright 2024 SUSE LCC
```